docs/reference/commands/index.asciidoc

```diff
@@ -1,26 +1,25 @@
-[role="xpack"]
-[[xpack-commands]]
-= {xpack} Commands
+[[commands]]
+= Commands
 
 [partintro]
 --
 
-{xpack} includes commands that help you configure security:
+{es} includes the following commands that help you configure security:
 
 * <<certgen>>
 * <<certutil>>
 * <<migrate-tool>>
-* <<saml-metadata>>
 * <<setup-passwords>>
 * <<syskeygen>>
+* <<saml-metadata>>
 * <<users-command>>
```

Comment:
Was this change intentional? I'd be inclined to list the items alphabetically as it's easier to maintain sanity when items are added.

Reply:
It wasn't alphabetical anymore, since those commands are now prefixed with "elasticsearch-"